llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -1661,6 +1663,19 @@ class ConstantPtrAuth final : public Constant {
   }
 };
 
+class ConstantExpr : public Constant {
+  ConstantExpr(llvm::ConstantExpr *C, Context &Ctx)
+      : Constant(ClassID::ConstantExpr, C, Ctx) {}
+  friend class Context; // For constructor.
+
+public:
+  /// For isa/dyn_cast.
+  static bool classof(const sandboxir::Value *From) {
+    return From->getSubclassID() == ClassID::ConstantExpr;
+  }
+  // TODO: Missing functions.
```

Comment:
Why not add the missing functions or a subset that is needed for now?  How is this useful?

Reply:
I think ConstExpr is being phased out gradually, or parts of it? I don't think we really need it for now.